src/r_main.c

```diff
@@ -642,6 +643,34 @@ static void R_SetupMouselook(fixed_t viewpitch)
   }
 }
 
+static inline angle_t LerpAngle(angle_t oldvalue, angle_t newvalue)
+{
+  return R_InterpolateAngle(oldvalue, newvalue, fractionaltic);
+}
+
+static inline fixed_t LerpFixed(fixed_t oldvalue, fixed_t newvalue)
+{
+  return (oldvalue + FixedMul(newvalue - oldvalue, fractionaltic));
+}
```

Comment:
I like these Lerp* functions, maybe we should use them everywhere. But it's for another PR/commit.